packages/react/src/Badge/Badge.tsx

```diff
@@ -7,24 +7,10 @@ import clsx from 'clsx'
 import { forwardRef } from 'react'
 import type { ForwardedRef, HTMLAttributes } from 'react'
 
-export const badgeColors = [
-  'black',
-  'blue',
-  'dark-green',
-  'green',
-  'grey-1',
-  'grey-2',
-  'grey-3',
-  'light-blue',
-  'magenta',
-  'orange',
-  'purple',
-  'red',
-  'white',
-  'yellow',
-] as const
+const defaultColor = 'dark-green'
```

Comment:
Same thing here as for Avatar: we don't need to add this default color to the list, it currently sets an unused class. We should add the default color to the Storybook controls though.